core/models/officeblock.py

```diff
@@ -34,3 +34,15 @@ class Meta:
 
     def __str__(self):
         return "{}".format(self.number)
+
+
+class FloorSection(models.Model):
```

Comment:
Rename the model to `OfficeFloorSection` and the fields as follows:
- `section_name` to `name`.
- `floor_number` to `floor`.

Reply:
Refactor all references to the model to reflect the rename.